src/browser/modules/D3Visualization/lib/visualization/components/visualization.ts

```diff
@@ -267,11 +269,16 @@ const vizFn = function(
     return latestStats
   }
 
-  viz.update = function() {
+  viz.update = function(options: {
+    updateNodes: boolean
```

Comment:
This method seems to not use the new options?

Reply:
`options` is passed into `geomotry.onGraphChange(graph, options)` in line 280.
https://github.com/FallingCeilingS/neo4j-browser/blob/visualisation-performance-optimisation-2/src/browser/modules/D3Visualization/lib/visualization/components/visualization.ts#L280

Reply:
ah, sorry It seems I've become blind